components/organisation-unit-tree/src/organisation-unit-node/organisation-unit-node.js

```diff
@@ -177,6 +191,7 @@ OrganisationUnitNode.propTypes = {
     dataTest: PropTypes.string.isRequired,
     id: PropTypes.string.isRequired,
     renderNodeLabel: PropTypes.func.isRequired,
+    rootId: PropTypes.string.isRequired,
```

Comment:
This addition should be a `feat` if it's part of the public API for the component.

Reply:
This is internal. The component we expose is the `OrganisationUnitTree`, which renders the `OrganisationUnitNode` and provides the `rootId` [here](https://github.com/dhis2/ui/blob/c05de2bbd90a3293af698917ec44d93e3423fd22/components/organisation-unit-tree/src/organisation-unit-tree/organisation-unit-tree.js#L84).

Reply:
Ah ! Great, thanks for clarifying.